### PR TITLE
Extract from go-libp2p

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
The master branch of this repo is a subtree split of go-libp2p: `git subtree split -P p2p/protocol/identify/`

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+os:
+  - linux
+  - osx
```
Comment (Kubuxu) [inline .travis.yml]: Please remove the osx, I am slowly removing it from most libp2p repos.

Comment (ghost) [reply .travis.yml]: updated

**Comment by Stebalien** [comment]
(out of date)

**Pull request closed without merging**